core/pkg/resultshandling/results.go

```diff
@@ -65,13 +65,19 @@ func (resultsHandler *ResultsHandler) GetResults() *reporthandlingv2.PostureRepo
 // HandleResults handle the scan results according to the pre defined interfaces
 func (resultsHandler *ResultsHandler) HandleResults() error {
 
-	resultsHandler.printerObj.ActionPrint(resultsHandler.scanData)
+	printerObjs := resultsHandler.printerObjs
+
+	for _, printerObj := range printerObjs {
```

Comment:
```suggestion
	for i := range printerObjs {
		printerObjs[i].ActionPrint(resultsHandler.scanData)
	}
```
I dont think we should iterate over the objects.

Reply:
Done. Made the suggested changes.